Flicker/Global/Utils/FirebaseManager.swift

```diff
@@ -253,4 +255,18 @@ final class FirebaseManager: NSObject {
             return ""
         }
     }
+    
+    // MARK: - storing Artist Data to the Database
+    func storeArtistInformation(_ artist: Artist) async {
+        guard let uid = auth.currentUser?.uid else { return }
+        /// 로그인시 UserDefaults 에 user 인포를 다 저장시키는데 이 기능이 구현되고 나면 Artist 에 user 필드 추가, 아직은 기능 노놉
+//        guard let userInfo = UserDefaults.d
+        do {
+            let artistData = ["state": artist.state, "regions": artist.regions, "camera": artist.camera, "lens": artist.lens, "detailDescription": artist.detailDescription, "portfolioImageUrls":  artist.portfolioImageUrls.sorted()] as [String : Any]
+            try await firestore.collection("artists").document(uid).setData(artistData)
+            print("⭐️⭐️⭐️URL UPLOAD DONE ⭐️⭐️⭐️")
+        } catch {
+            print("error string Artist Model")
```

Comment:
p3

catch문 이하에는 스트링에 error를 같이 넣어주면 어떨까요?
```suggestion
            print("error string Artist Model \(error)")
```

try가 앞에 붙은 곳은 코드를 실행하면서 만약 에러가 있다면! error를 throw하게 되는데 catch에서 이를 잡아서 출력할 수 있습니다! 뭐가 잘 안된다면 디버깅하기 용이합니다!